### ISSUE TITLE
Closure Linter 2.3.5 does not work with non-breakable whitespace character 0xA0

### ISSUE THREAD

**Issue by GoogleCodeExporter** [google/closure-linter#49]
```
What steps will reproduce the problem?
1. Use a non-breakable whitespace character in your code

What is the expected output? What do you see instead?

The expected output should be: a warning or it should simply work.
Instead I see:
Line 9, E:-002: Error parsing file at token "}". Unable to check the rest of 
file.

The linenumber does not match the linenumber of the non-breaking whitespace


What version of the product are you using? On what operating system?
gjslint 2.3.5, Mac OSX 10.7.4


Additional Info:
The attached file has a non-breaking whitespace after the keyword "function".
Closure Linter does not break, if function block is empty.



```

Original issue reported on code.google.com by `mail.jan...@gmail.com` on 2 Jul 2012 at 2:42

Attachments:
- [invalidCharacter.js](https://storage.googleapis.com/google-code-attachments/closure-linter/issue-49/comment-0/invalidCharacter.js)
